server/bigbluebuttonapiwrapper/api/APIcalls.go

```diff
@@ -80,9 +81,17 @@ func CreateMeeting(meetingRoom *dataStructs.MeetingRoom) (string, error) {
 		url.QueryEscape(meetingRoom.Meta_endcallbackurl)
 	voiceBridge := "&voiceBridge=" + url.QueryEscape(meetingRoom.VoiceBridge)
 
+	metaBBBOrigin := "&meta_bbb-origin=bbb-mm" +
+		"&meta_bbb-origin-version=" + url.QueryEscape(helpers.PluginVersion)
+
+	hostname, err := os.Hostname()
```

Comment:
The operating system's hostname is not always unique. For example with containters or multiple mattermost instances on the same server. Wouldn't it be more reliable to take the serverURL from the mattermost configuration?

Reply:
Absolutely!